crypto/cloudkms/signer.go

```diff
@@ -0,0 +1,63 @@
+package cloudkms
```

Comment:
I would place this in the models package, no reason to create a new package for it

Reply:
The current `KeyRotationSigner` is under the `requester` package, and it seems like a good place to put the Cloud KMS rotation signer as well. It isn't a model, strictly speaking, such as a `Block` / `Transaction`/ `Receipt` / `Event` etc.
In https://github.com/onflow/flow-evm-gateway/pull/282/commits/f9303971f9c0a580a52904b3cd8e33edef785b53, I have moved it to the `requester` package.